### DISCUSSION TITLE
Баг. Не запускается приложение после закачивания карт

### DISCUSSION THREAD

**Discussion by Condorillo** [andreynovikov/trekarta#179] [General]
Смартфон POCO X3 NFC, MIUI Global 12.5.1, Android 11 RKQ1.200826.002.
Первый запуск после установки приложения проходит нормально. Как только закачиваешь каку-нибудь карту — всё, приложение перестает запускаться, виснет на белом экране с иконкой приложения.
Об этой проблеме писал на почту, ответа не получил.

**Comment by andreynovikov** [comment]
Попробуйте новую версию (2022.04)
